.eslintrc.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+    "env": {
+        "browser": true,
+        "es2020": true,
+        "jquery": true
+    },
+    "extends": "eslint:recommended",
+    "parserOptions": {
+        "ecmaVersion": 11,
+        "sourceType": "module"
+    },
+    "rules": {
+    }
+};
```

Comment:
Use the same config as the one proposed by Microverse [ see [config](https://github.com/microverseinc/linters-config/blob/master/javascript/.eslintrc.json) ]